File: 10079.cpp
/*c024: 10079 - Pizza Cutting*/
#include<iostream>
using namespace std;

int main(){
    int n;
    while(cin>>n&&n>0){
        if(n==1){
            cout<<2<<endl;
        }else{
            long long t=2;
            for(int i=2;i<=n;i++){
                t+=i;
            }
            cout<<t<<endl;
        }
    }
    return 0;
}

File: 10079.py
''' c024: 10079 - Pizza Cutting '''
import numpy as np

A = np.zeros((5))
A[0] = 3
A[1] = 9
print(A)

n = 1498
temp = n + A[0]
i = 0
while(temp>=10):
    A[i] = temp%10
    temp //= 10
    i += 1

print(A)